### PR TITLE
feat: Secondary 2 no pic and 2 Slice

### PR DESCRIPTION
**Pull request by aronshamash**
- new slice in `slice-layout` (no margins in the layout)
- new Tile G  in `edition-slices` (margins in the tile)
- new image crop in `fixture-generator`
- https://nidigitalsolutions.jira.com/browse/REPLAT-5143

### Slice Layout
screenshot 2019-02-05 at 11 13 11

### Tile G
screenshot 2019-02-05 at 11 13 35

### Edition Slice
screenshot 2019-02-05 at 11 13 54

### CODE REVIEW

**Comment by nehasri89** [comment]
can you put some screenshot pls @aronshamash

**Comment by tuncaulubilge** [comment]
What's the problem with images on the screenshots? Even with fixture generator, we should be seeing some images on storybooks

File: packages/edition-slices/src/tiles/tile-b/styles/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { fonts, spacing } from "@times-components/styleguide";
 
 const styles = {
+  container: {
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/tile-b/styles/index.js]: I'm confused why this is not reflected on the snapshots

File: packages/edition-slices/src/tiles/tile-g/styles/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  headline: {
+    ...fontFactory({
+      font: "headline",
+      fontSize: "infoTitle"
+    })
+  },
+  imageContainer: {
+    borderColor: colours.functional.contrast,
+    borderRadius: Dimensions.get("window").width / 4,
+    marginLeft: "auto",
```
[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply packages/edition-slices/src/tiles/tile-g/styles/index.js]: Let's merge that one in and see :)

**Pull request merged**